src/mgmt/srv_pool.c

```diff
@@ -110,13 +110,26 @@ ds_mgmt_tgt_pool_destroy(uuid_t pool_uuid)
 	return DER_SUCCESS;
 }
 
+static uint32_t
+pool_tgt_create_timeout(void)
+{
+	unsigned int t = 60;
+
+	d_getenv_int("CRT_TIMEOUT", &t);
+	if (t == 0)
+		t = 60;
+	d_getenv_int("DAOS_POOL_CREATE_TIMEOUT", &t);
```

Comment:
if DAOS_POOL_CREATE_TIMEOUT=0 in the environment, suggest that the timeout should be whatever the value of t was before this d_getenv_int() call.

Reply:
Sure.

Reply:
any reason to re-query env each time instead of saving this info once? env isnt going to change from call to call

Reply:
No particular reason; when I wrote this patch I wasn't sure if we'd want to land it, so I did it quick and dirty.